clang/lib/Frontend/FrontendAction.cpp

```diff
@@ -997,8 +997,9 @@ bool FrontendAction::BeginSourceFile(CompilerInstance &CI,
   std::optional<cas::IncludeTreeRoot> IncludeTreeRoot;
   std::optional<StringRef> IncludeTreePCHBuffer;
   if (Input.isIncludeTree()) {
-    if (llvm::Error E = cas::IncludeTreeRoot::get(CI.getOrCreateObjectStore(),
-                                                  Input.getIncludeTree())
+    llvm::cas::ObjectStore &DB = CI.getOrCreateObjectStore();
+
+    if (llvm::Error E = cas::IncludeTreeRoot::get(DB, Input.getIncludeTree())
```

Comment:
Is this change needed?

Reply:
No, must be a leftover. I'll remove this.